components/html/modules/content-newsletter-text-only/_main-1.html

```diff
@@ -0,0 +1,49 @@
+<section class="intro">
+	<article>
+		<h1 class="intro__title">Title of Enewsletter Here</h1>
+			<p class="intro__issue">FALL 2017</p>
+			
+			<p class="intro__description">A brief introduction to the newsletter can go here if needed. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis vitae lacinia dui, ac lacinia dolor. Maecenas mattis nulla ac purus mattis, sit amet facilisis dolor luctus. Donec luctus convallis arcu.</p>
+	</article>
+</section>
+		
+<section class="section">
+	<article>
+		<h2 class="section__heading">Featured Stories</h2>
+		
+			<section class="story">
+				<h3 class="story__title">This title is for the featured story. Three lines used if necessary or needed.</h3>
+				<p class="story__subtitle">Subhead if needed</p>
+				<p class="story__teaser">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis vitae lacinia dui, ac lacinia dolor. Maecenas mattis nulla ac purus mattis, sit amet facilisis dolor luctus. Donec luctus convallis arcu. Nullam adipiscing, magna porttitor pellentesque ornare, magna tortor dignissim justo, vel fermentum massa neque sit amet urna.</p>
+			
+				<a href="#" class="button button--hover-light">Read More</a>
+			</section>
+	</article>
+</section>
+		
```

Comment:
The list section needs updated class names and structured based on the code I sent you earlier.